tools/refresh-keyrings.sh

```diff
@@ -45,11 +29,40 @@ ESM_KEY_ID_BIONIC="2926E7D347A1955504000A983121D2531EF59819"
 # fips and fips-updates are same key ID
 FIPS_KEY_ID_XENIAL="E23341B2A1467EDBF07057D6C1997C40EDE22758"
 
+generate_keyrings() {
+    KEYRING_DIR="$1"
+    shift
+    KEYS="$@"
+
+    # Intentionally unquoted so GPG gets the keys as separate arguments
+
+    for key in $KEYS; do
+        case $key in
+            $EAL_KEY_ID_XENIAL)
+                service_name="Common-Criteria-EAL2";;
+            $ESM_KEY_ID_TRUSTY|$ESM_KEY_ID_XENIAL|$ESM_KEY_ID_BIONIC)
+                service_name="ESM-For-Infrastructure";;
+            $FIPS_KEY_ID_XENIAL)
+                service_name="FIPS";;
+        esac
+        keyring_file="$KEYRING_DIR/Ubuntu-Advantage-$service_name-$key.gpg"
```

Comment:
Title-Case is inconsistent with all the lower-case keyrings already in `/usr/share/keyrings`

Reply:
used lowercase-human-readable  now.